pkgs/development/libraries/dbus/default.nix

```diff
@@ -15,12 +16,19 @@
 , docbook_xml_dtd_44
 , docbook-xsl-nons
 , xmlto
+, autoreconfHook
+, autoconf-archive
 }:
 
 stdenv.mkDerivation rec {
   pname = "dbus";
   version = "1.12.20";
 
+  ## WARNING when updating:
+  # AC_PATH_XTRA patch below should be included
```

Comment:
We can make it unconditional, now that it was merged. That way updating will make it obvious the patch no longer applies.

Reply:
Should be fixed now, though I didn't had to time the build because it triggered a massive rebuild.